### PR TITLE
chore(ref-imp): add controller back into verification methods

### PR DESCRIPTION
**Pull request by isaacJChen**

### CODE REVIEW

File: tests/vectors/resolution/afterCreate.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "verificationMethod": [
       {
         "id": "#publicKeyModel1Id",
+        "controller": "",
```
Comment (troyronda) [inline tests/vectors/resolution/afterCreate.json:21]: Controller should not be an empty string. Prefer to omit property rather than have an empty string.

**Review by troyronda — CHANGES_REQUESTED** [review]

File: tests/vectors/resolution/afterCreate.json (hunk 1 above)
Comment (isaacJChen) [reply tests/vectors/resolution/afterCreate.json:21]: It has to be there according to did-core spec
image

Comment (isaacJChen) [reply tests/vectors/resolution/afterCreate.json:21]: We have a base in @context, which fills it in with JSON-LD

Comment (troyronda) [reply tests/vectors/resolution/afterCreate.json:21]: I see.

Note: personally, I think this style makes the resolution result less clear to the human reader (and also by simpler processors.)

Comment (isaacJChen) [reply tests/vectors/resolution/afterCreate.json:21]: Ya, I agree. I removed it in my last PR based on WG discussion results. We can further discuss it if we want the plain DID in there and not infer it separately. Having it there as empty string at least is spec compliant and was the previous state so I decided to revert it. Feel free to bring this up in the next WG meeting though :)

**Review by thehenrytsai — APPROVED** [review]

File: tests/vectors/resolution/afterCreate.json (hunk 1 above)
Comment (thehenrytsai) [reply tests/vectors/resolution/afterCreate.json:21]: @troyronda, this is a follow up PR for discussion that occurred in https://github.com/decentralized-identity/sidetree/pull/1006, thus @sandrask is aware of this change and I believe we have consensus, thus I am approving this PR to unblock.

Comment (troyronda) [reply tests/vectors/resolution/afterCreate.json:21]: @thehenrytsai I have opened a followup https://github.com/decentralized-identity/sidetree/issues/1010 as I do not think using an empty string is the best outcome.

**Pull request merged**